### DISCUSSION TITLE
Lotw downloads but no update ?

### DISCUSSION THREAD

**Discussion by ceb77** [foldynl/QLog#317] [Q&A]
Hello,
Wonderful program so far but when I download QSO's from Lotw the program shows 373 but 0 updated. I am assuming that the log on LOTW will sync with the local QLog but I cant seem to make it do that. I have the correct path to the TQSL file. I haven't tried an upload and signing a log yet . Any help ?
Thanks,
Eddie W4HL

**Comment by foldynl** [comment]
Please, could you send me a screenshot of the Download dialog?

**Comment by ceb77** [comment]
Hello,
![Screenshot from 2024-02-13 08-45-25](https://github.com/foldynl/QLog/assets/111024356/3f93ba97-b768-4f75-900d-3a620dd7772f)

then
![Screenshot from 2024-02-13 08-42-13](https://github.com/foldynl/QLog/assets/111024356/ae016cf1-2f8f-42be-bfd5-8aa1592772d5)

But this download is not added to the local log.

I can download a report from lotw ,it is an adi file of confirmed QSLs, and it can be imported fine. But it is only a list confirmed QSLs.

Something else I noticed is that the Lotw dialog box will not show your own call if the local log is blank. If you save one call then your own call will then show in the box.

Thanks very much for your work on this project. 
Thanks,
Eddie, W4HL

**Comment by foldynl** [comment]
I understand the problem. There was probably a misunderstanding. LoTW download is only used to download QSLs from LoTW. That means that you must have the QSO in QLog and you can only download the confirmation (QSL) from LoTW. 

I understand LoTW as something like "a contact verification service" instead of "being a data synchronization platform". Among other things, this is also due to the fact that the downloaded QSOs from LoTW contain only a limited set of QSO parameters.

Therefore, I would advise you to first import full QSO records using the QLog import and then download the QSLs from LOTW. 

Thanks for opening this issue. I think I should write it there that it is not a QSO sync but only QSL download.

**Reply by ceb77** [reply]
Yes, Thank You,
I was able to download a full QSO adi file from LOTW using the http query. I then imported the file into local log successfully.
Thanks
Eddie, W4HL
